insights/specs/__init__.py

```diff
@@ -146,6 +146,7 @@ class Specs(SpecSet):
     etc_journald_conf = RegistryPoint()
     etc_machine_id = RegistryPoint()
     etc_udev_40_redhat_rules = RegistryPoint(filterable=True)
+    etc_udev_oracle_asm_rules = RegistryPoint(filterable=True)
```

Comment:
@qinpingli this spec is a `glob_file` which means that it can collect multiple inputs, so you'll need to add the attribute `multi_output=True`.

Reply:
@bfahr Good catch, thanks! Updated.